playground/frontend/playground_components/tools/extract_symbols_java/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
```

Comment:
This looks like KTS in a file named to be in Groovy.

Reply:
`./gradlew rat`

Reply:
done

Reply:
deleted